recipes/qxlsx/all/conanfile.py

```diff
@@ -25,34 +26,42 @@ class QXlsxConan(ConanFile):
         "fPIC": True
     }
 
+    @property
+    def _qt_version(self):
+        return Version(self.deps_cpp_info["qt"].version).major
+
     def export_sources(self):
-        for p in self.conan_data.get("patches", {}).get(self.version, []):
-            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
 
     def requirements(self):
-        self.requires("qt/5.15.5")
+        self.requires("qt/5.15.7")
 
     def layout(self):
-        cmake_layout(self)
+        cmake_layout(self, src_folder="src")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/3.25.0")
```

Comment:
Why is it required unconditionally while it was building fine in c3i for older versions?

Reply:
Sorry you already review this here: https://github.com/conan-io/conan-center-index/pull/14562#discussion_r1039379016

I just add the check here: 4f8882579